### PR TITLE
core/cmd_analysis: use API instead of calling rz_core_cmd

### PR DESCRIPTION
**Pull request by ret2libc**
**DO NOT SQUASH**

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Use API instead of calling rz_core_cmd0

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: librz/core/cmd_analysis.c [hunk 1]
```diff
@@ -7775,144 +7626,193 @@ static void cmd_agraph_node(RzCore *core, const char *input) {
...
 		break;
-	}
 	case '?': // "age?"
 	default:
 		rz_core_cmd_help(core, help_msg_age);
 		break;
 	}
 }
 
+RZ_IPI void rz_core_agraph_print_custom(RzCore *core) {
```
Comment (XVilka) [inline librz/core/cmd_analysis.c]: I suggest putting those into the separate file, e.g. `librz/core/cagraph.c` or something like that.

**Review by XVilka — CHANGES_REQUESTED** [review]

**Comment by XVilka** [comment]
Uh, currently `librz/core/agraph.c` is the visual ASCII graph interface. So the IPI is a bit out of the place here. What about this:
- Rename current `librz/core/agraph.c` into `librz/core/vgraph.c`
- Move these IPI `rz_core_agraph_*` functions into the newly created `librz/core/agraph.c`?

P.S. Please also fix the build.

**Review by wargio — APPROVED** [review]

**Comment by ret2libc** [comment]
> Uh, currently `librz/core/agraph.c` is the visual ASCII graph interface. So the IPI is a bit out of the place here. What about this:
> 
> * Rename current `librz/core/agraph.c` into `librz/core/vgraph.c`
> * Move these IPI `rz_core_agraph_*` functions into the newly created `librz/core/agraph.c`?
> 
> P.S. Please also fix the build.

I can create `cagraph.c` actually. I  think renaming agraph to vgraph might be confusing, because agraph.c contains all the agraph functions AFAIR.

**Comment by XVilka** [comment]
Clang complains with:
```
Check failure on line 19 in librz/core/cagraph.c

Code scanning

Potential leak of memory pointed to by 'b'
```

**Review by XVilka — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**